ckan/plugins/interfaces.py

```diff
@@ -1492,7 +1492,7 @@ class IUploader(Interface):
     upload resources and group images.
     '''
 
-    def get_uploader(self):
+    def get_uploader(self, upload_to, old_filename):
```

Comment:
The signature here was actually wrong.
When get_uploader is called on plugins, it's actually passed one arguments
see
https://github.com/ckan/ckan/blob/ed2196b31421365fc559eb8e0fa9141fa17a057f/ckan/lib/uploader.py#L24